docs/commands.md

```diff
@@ -0,0 +1,65 @@
+---
+id: commands
+title: Command Line Interface Reference
+sidebar_label: CLI Reference
+---
+
+This page is a quick reference for all of the command that are used to manage the application.
+
+## Setup Command
+
+The setup command loads the website configuration and creates the database. It can only be run once (unless the database
+is manually deleted). If you need to change a running system then you will need to use the reset command instead.
+
+The command is executed by typing:
+
+```bash
+flask --debug setup [path_to_configuration]
+```
+
+## Reset Command
+
+The reset command reloads the website configuration and resets the database after the setup command has been run.
+
+This command can be used when:
+
+1. The website configuration file has been modified.
+1. The content of the database needs to be deleted and reset.
```

Comment:
Does this need to be '2.' instead of '1.' - or does this get rendered as a sequential list just by having any number put here?

Reply:
No the display sorts that out which means no renumbering if you add something later